ui/component/core/src/index.ts

```diff
@@ -981,11 +1096,22 @@ export class Manager implements EventProviderFactory {
     }
 
     protected async updateKeycloakAccessToken(): Promise<boolean | void> {
+
```

Comment:
We should set token refresh based on keycloak settings, maybe create an issue for this